docs/modules/system preparation.rst

```diff
@@ -0,0 +1,309 @@
+.. _system_preparation:
+
+Preparing systems for constant-pH simulation
```

Comment:
Can we at least split off a sensible stable API for supporting amber input files then? Otherwise I am not sure how moving forward will not simply break everything